backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -266,7 +294,75 @@ export class FeatureFlagService {
     featureFlag.id = flagDTO.id;
     featureFlag.key = flagDTO.key;
     featureFlag.status = flagDTO.status;
+    featureFlag.context = flagDTO.context;
+    featureFlag.tags = flagDTO.tags;
+    const newExclusion = new FeatureFlagSegmentExclusion();
+    const newInclusion = new FeatureFlagSegmentInclusion();
+    featureFlag.featureFlagSegmentExclusion = { ...flagDTO.featureFlagSegmentExclusion, ...newExclusion };
```

Comment:
Should not this be 
`{ ...newExclusion, ...flagDTO.featureFlagSegmentExclusion }`?

Reply:
I'm not sure it matters because `newExclusion` isn't being instantiated with any attributes, but I'll reverse those spreads.